java/pom.xml

```diff
@@ -151,6 +155,8 @@
                 </executions>
                 <configuration>
                     <redirectTestOutputToFile>true</redirectTestOutputToFile>
+                    <forkCount>1</forkCount>
```

Comment:
POM changes aren't necessary.  I was trying to resolve UnsatisfiedLinkErrors and never changed it back.  Will fix.



On Oct 2, 2015, 7:41 AM, at 7:41 AM, Jon Morra <notifications@github.com> wrote:
>> @@ -151,6 +155,8 @@
>>                  </executions>
>>                  <configuration>
>>                     
><redirectTestOutputToFile>true</redirectTestOutputToFile>
>> +                    <forkCount>1</forkCount>
>
>@deaktator is this really necessary?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/marko-asplund/vowpal_wabbit/pull/3/files#r41028244